controllers/release/adapter.go

```diff
@@ -478,3 +492,43 @@ func (a *Adapter) syncResources() error {
 
 	return a.syncer.SyncSnapshot(snapshot, releasePlanAdmission.Namespace)
 }
+
+// registerAttributionData updates the status of the Release being processed with the proper attribution author.
+func (a *Adapter) registerAttributionData(releasePlan *v1alpha1.ReleasePlan) error {
+	if a.release.IsAttributed() {
+		return nil
+	}
+
+	var author string
+	patch := client.MergeFrom(a.release.DeepCopy())
+
+	if a.release.Labels[metadata.AutomatedLabel] == "true" {
+		author = releasePlan.Labels[metadata.AuthorLabel]
+		if author == "" {
+			return fmt.Errorf("no author in the ReleasePlan found for automated release")
+		}
+		a.release.Status.Attribution.StandingAuthorization = true
+	} else {
+		author = a.release.Labels[metadata.AuthorLabel]
+		if author == "" { // this should be impossible
+			return fmt.Errorf("no author found for manual release")
+		}
+	}
+	a.release.Status.Attribution.Author = author
```

Comment:
I think I commented this already, but you don't need to declare an `author` variable. Just use `a.release.Status.Attribution.Author` directly. One less assignment as you don't need this line. If you really want to go this way, add an empty line before this one please.

Reply:
Not sure I follow. I set `author` in the if statement based on conditions, then set the value in the status to whatever I found in the if statement. I can add an empty line though

Reply:
I was just saying that you don need the `author` variable in this block. That you can use the target directly (`a.release.Status.Attribution.Author`) but I'm fine with this.